src/webview/js/js.js

```diff
@@ -220,7 +220,8 @@ const scrollToPreserve = (msgId: number, prevBoundTop: number) => {
 
 const appendAuthToImages = auth => {
   const imageTags = document.getElementsByTagName('img');
-  Array.from(imageTags).forEach(img => {
+  const imageTagsArray = Array.prototype.slice.call(imageTags);
```

Comment:
This is a neat trick!

I feel like it calls for some explanation, or at least some isolation as a thing with a name; it's a bit puzzling at first look. ("Why isn't this reduced to just `imageTags.slice()`?)

Perhaps a function named `arrayFrom`? Perhaps also with a link to this handy section on the MDN page, about exactly this trick:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#Array-like_objects